static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -157,6 +165,61 @@ define([
         return tag.join(',');
     }
 
+    function abData(variantName, complete) {
+        return {
+            'variantName': variantName,
+            'complete': complete
+        };
+    }
+
+    function getAbLoggableObject() {
+        try {
+            return reduce(zip(getActiveTests(), getServerSideTests()), function(log, tests) {
```

Comment:
nah, generally server side tests will be a lot smaller but `zip` will continue for the length of the longest array, adding `undefined` if there's nothing in the other one. that's why i had to add the check for `server` on line 189. 

thinking about it i should add an equivalent check for `active` to cover the unlikely event that there are more server than client-side tests. not a silly question at all!